cmd/init.go

```diff
@@ -44,7 +44,7 @@ func init() {
 
 	// set flags of build command
 	buildCmd.Flags().StringVarP(&buildCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as the default")
-	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", "The path to the results folder; the current path is set as the default")
+	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", "The path to the results folder; If this parameter is not provided, the MTAR file is saved in the mta_archives subfolder of the current folder")
```

Comment:
Either the `I` in `If` should be small or there should be a `.` instead of `;` before it